pyproject.toml

```diff
@@ -121,14 +124,25 @@ prerelease = "if-necessary-or-explicit"
 index-strategy = "unsafe-best-match"                                    # Needed for TRT-LLM
 
 [tool.uv.sources]
-torch = [{ index = "pytorch-nightly-cu128" }]
-torchvision = [{ index = "pytorch-nightly-cu128" }]
+torch = [
+    { index = "pytorch-nightly-cu128", marker = "platform_machine != 'aarch64' or (platform_machine == 'aarch64' and 'tegra' not in platform_release)" },
+    { index = "jetson-containers", marker = "platform_machine == 'aarch64' and 'tegra' in platform_release" },
+]
+torchvision = [
+    { index = "pytorch-nightly-cu128", marker = "platform_machine != 'aarch64' or (platform_machine == 'aarch64' and 'tegra' not in platform_release)" },
+    { index = "jetson-containers", marker = "platform_machine == 'aarch64' and 'tegra' in platform_release" },
+]
 
 [[tool.uv.index]]
 name = "pytorch-nightly-cu128"
 url = "https://download.pytorch.org/whl/nightly/cu128"
 explicit = false
 
+[[tool.uv.index]]
+name = "jetson-containers"
+url = "https://pypi.jetson-ai-lab.dev/jp6/cu126"
```

Comment:
There's a page : https://pypi.jetson-ai-lab.dev/jp6/cu128 
should this be cu128 ?